tests/framework/integration/cluster.go

```diff
@@ -543,6 +544,7 @@ type Member struct {
 	config.ServerConfig
 	UniqNumber                     int
 	MemberNumber                   int
+	Port                           string
```

Comment:
In what cases port is not a int? Could we just cast port number to int?

Reply:
in the unix socket case, there it's the name (eg m0). It's weird, I also started with an integer, but almost all stdlib methods are also using string for the port - so I ended up fmt.Sprintf all over the place.